easybuild/easyconfigs/b/Bowtie/void2int.patch

```diff
@@ -0,0 +1,13 @@
+--- processor_support.h.orig	2017-02-02 21:48:42.436432756 +0000
```

Comment:
@masaofujinaga please include some additional information at the top:

* where does the patch come from (self-made or upstream?); provide a link if appropriate)
* what's the purpose of the patch
* who's the author (include your contact information if you're the author)